frontend/src/views/dev_frame/load-mock-data.tsx

```diff
@@ -145,7 +151,7 @@ export function SeedDataMenu({
         setProgress(100);
     }
 
-    async function seedUsers(limit = 1000) {
+    async function seedUsers(limit = 1053) {
```

Comment:
I think we can cut this down to 600. That should speed things up but also be realistic.